scripts/cnv_wdl/somatic/cnv_somatic_allele_fraction_pair_workflow.wdl

```diff
@@ -0,0 +1,262 @@
+# Subworkflow for running GATK ACNV on tumor/normal or tumor-only cases. Supports both WGS and WES.
+#
+# Notes:
+#
+# - A normal BAM (normal_bam) is requrired for the tumor/normal workflow.  If not provided, the tumor-only workflow
+#   will be run.
+#
+# - ACNV is run only on the tumor.
```

Comment:
ACNV is run only on the tumor.  CNV is run on both the tumor and the normal, since this is how we did it before.  We could change this to be more parallel if you like (i.e., either run ACNV on both, or run CNV only on the tumor).